### ISSUE TITLE
Window navigation bar colour contrast issue

### ISSUE THREAD

**Issue by MaximPerry** [MaximPerry/Desktop-Timer#5]
The window title color and the background colour fails WCAG colour contrast criteria.

**Comment by MaximPerry** [comment]
Adapted colours to be WCAG compliant. Closing issue.
